f5_sphinx_theme/head.html

```diff
@@ -25,12 +25,15 @@
   <meta name="version" content="{{ release|striptags|e }}"/>
   <meta name="updated_date"  content="{{ last_updated|e }}"/>
   <meta name="archived" content="Archived documents excluded"/>
-
+{% if (theme_version_selector != 'False') %}
```

Comment:
This is nitpicky, but I'd like to separate these from the metadata above, which are required for federated search. Would you please add a comment here?